### PR TITLE
Feature#36 complete bottom tab navigation

### PR DESCRIPTION
**Pull request by EusebioAjas**
# Description
 
 Examples:
 - What?: I've added icons for bottom tab navigation.
 - Why?: These changes complete the navigation design provided in the issue #36 

  ## Screenshots 
image

### CODE REVIEW

**Review by Shadic78 — COMMENTED** [review]
Could you change the colors for the black and white from the design and add a bit of padding bottom?

**Comment by EusebioAjas** [comment]
> Could you change the colors for the black and white from the design and add a bit of padding bottom?

Done

**Review by OswaldoPineda — DISMISSED** [review]

File: GamerpotFrontend/src/library/constants/colors.ts [hunk 1]
```diff
@@ -6,4 +6,5 @@ export const colors = {
   darkGray: '#3B3B3B',
   dealsBackground: '#161618',
   green: '#329C30',
+  tabBackground: '#161618',
```
Comment (YilKanda) [inline GamerpotFrontend/src/library/constants/colors.ts:9]: Why are you using the same color in two different variables? You can implement colors, based on which is the primary, the secondary, the tertiary (according to the importance of it)

File: GamerpotFrontend/src/library/constants/tabScreens.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+} from '../../screens';
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  tabBackground: {
+    backgroundColor: colors.tabBackground,
+    paddingBottom: 4,
+  },
+});
+
```
Comment (YilKanda) [inline GamerpotFrontend/src/library/constants/tabScreens.ts:16]: You can export this section to a file for styles.

File: GamerpotFrontend/src/library/constants/colors.ts (hunk 1 above)
Comment (EusebioAjas) [reply GamerpotFrontend/src/library/constants/colors.ts:9]: No, I did not notice that it already exists

File: GamerpotFrontend/src/library/constants/tabScreens.ts (hunk 2 above)
Comment (EusebioAjas) [reply GamerpotFrontend/src/library/constants/tabScreens.ts]: Done

File: GamerpotFrontend/src/library/constants/colors.ts (hunk 1 above)
Comment (EusebioAjas) [reply GamerpotFrontend/src/library/constants/colors.ts]: `tabBackground` has been removed and replaced by dealsBackground.

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by Shadic78 — APPROVED** [review]

**Pull request merged**